test/e2e/apps/BUILD

```diff
@@ -61,6 +61,7 @@ go_library(
         "//staging/src/k8s.io/client-go/tools/cache:go_default_library",
         "//staging/src/k8s.io/client-go/tools/watch:go_default_library",
         "//staging/src/k8s.io/client-go/util/retry:go_default_library",
+        "//staging/src/k8s.io/kubectl/pkg/util/podutils:go_default_library",
```

Comment:
I'm not sure picking up the k8s.io/kubectl dep is a good idea, I couldn't find any prior art in test/e2e for importing from it.  Is there something in test/e2e/framework/pod/ we could use?

Reply:
@spiffxp That was a mistake. I have updated the PR to use the right library now.